opentreemap/otm_comments/tests.py

```diff
@@ -155,7 +155,7 @@ def test_sorting(self):
 
 
 def _comment_ids_to_params(*args):
-    return {'comment-ids': ','.join(str(arg) for arg in args)}
+    return {'ids': ','.join(str(arg) for arg in args)}
```

Comment:
curious about this rename. Based on it's uses/changes in this PR, it looks like the `comment-` prefix is useful for context.